src/biz/bokhorst/xprivacy/XPrivacy.java

```diff
@@ -29,64 +28,107 @@
 	private static String mSecret = null;
 	private static List<String> mListHookError = new ArrayList<String>();
 	private static List<CRestriction> mListDisabled = new ArrayList<CRestriction>();
-
-	// http://developer.android.com/reference/android/Manifest.permission.html
-
-	static {
-		if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP && mListDisabled.size() == 0) {
```

Comment:
Okay, it did not harm on my device so I left it enabled. But if it causes problems for some, then... Not really sure what it does anyways, I am guessing that it allows the app to pre-disable features from boot?